### PR TITLE
Changes for Cordova Sample App

### PR DESCRIPTION
**Pull request by kchitalia**

### CODE REVIEW

File: SalesforceSDK/Salesforce.SDK.SmartStore/Source/Util/Constants.cs [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+        #region Salesforce object type layout field constants.
+
+        public const string LayoutLimitsField = "limitRows";
+        public const string LayoutColumnsField = "searchColumns";
+
+        #endregion
+
+        #region SyncState, SyncOptions, Status constants
+
+        public const string SyncsSoup = "syncs_soup";
```
Comment (bhariharan) [inline SalesforceSDK/Salesforce.SDK.SmartStore/Source/Util/Constants.cs]: Sync related constants and other sync related stuff is probably better suited to ```SmartSync``` than ```SmartStore```.

File: SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/phone/bld/ProjectArchitectures.txt [hunk 2]
```diff
@@ -1 +1 @@
-C:\Repo\sdk-github\SalesforceSDK\Salesforce.Sample.Cordova.AccountManager\platforms\windows\CordovaApp.Phone.jsproj;neutral
+C:\dev\kunal\SalesforceMobileSDK-Windows\SalesforceSDK\Salesforce.Sample.Cordova.AccountManager\platforms\windows\CordovaApp.Phone.jsproj;x86
```
Comment (bhariharan) [inline SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/phone/bld/ProjectArchitectures.txt]: Hardcoded directory path?

File: SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/windows/bld/SQLitePCL/Resources.resfiles [hunk 3]
```diff
@@ -0,0 +1 @@
+C:\dev\kunal\SalesforceMobileSDK-Windows\SalesforceSDK\Salesforce.Sample.Cordova.AccountManager\platforms\windows\build\windows\bld\SQLitePCL\en\SQLitePCL.Resources.resw
```
Comment (bhariharan) [inline SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/windows/bld/SQLitePCL/Resources.resfiles]: Hardcoded path.

File: SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/windows/bld/SQLitePCL/Resources.resfiles.intermediate [hunk 4]
```diff
@@ -0,0 +1 @@
+C:\dev\kunal\SalesforceMobileSDK-Windows\SalesforceSDK\Salesforce.Sample.Cordova.AccountManager\platforms\windows\build\windows\bld\SQLitePCL\en\SQLitePCL.Resources.resw
```
Comment (bhariharan) [inline SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/windows/bld/SQLitePCL/Resources.resfiles.intermediate]: Hardcoded path.

File: SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/windows/bld/pri.resfiles [hunk 5]
```diff
@@ -1 +1,5 @@
+C:\dev\kunal\SalesforceMobileSDK-Windows\SalesforceSDK\Salesforce.Sample.Cordova.AccountManager\platforms\windows\build\windows\bld\SQLitePCL.pri
```
Comment (bhariharan) [inline SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/windows/bld/pri.resfiles]: Hardcoded path.

File: SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/windows/bld/pri.resfiles.intermediate [hunk 6]
```diff
@@ -1 +1,5 @@
+C:\dev\kunal\SalesforceMobileSDK-Windows\SalesforceSDK\Salesforce.Sample.Cordova.AccountManager\platforms\windows\build\windows\bld\SQLitePCL.pri
```
Comment (bhariharan) [inline SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/windows/bld/pri.resfiles.intermediate]: Hardcoded path.

File: SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/phone/bld/ProjectArchitectures.txt (hunk 2 above)
Comment (kchitalia) [inline SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/phone/bld/ProjectArchitectures.txt]: I'll add these files to .gitignore. We dont need them checked in

File: SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/windows/bld/SQLitePCL/Resources.resfiles (hunk 3 above)
Comment (kchitalia) [inline SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/windows/bld/SQLitePCL/Resources.resfiles]: Same here

File: SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/windows/bld/SQLitePCL/Resources.resfiles.intermediate (hunk 4 above)
Comment (kchitalia) [inline SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/windows/bld/SQLitePCL/Resources.resfiles.intermediate]: Same here

File: SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/windows/bld/pri.resfiles (hunk 5 above)
Comment (kchitalia) [inline SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/windows/bld/pri.resfiles]: Same here

File: SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/plugins/com.salesforce/tools/update.ps1 [hunk 7]
```diff
@@ -110,7 +110,7 @@ $ANDROID_SDK_REPO_PATH="https://github.com/forcedotcom/SalesforceMobileSDK-Andro
 $ANDROID_SDK_FOLDER="SalesforceMobileSDK-Android"
 $SHARED_SDK_REPO_PATH="https://github.com/forcedotcom/SalesforceMobileSDK-Shared.git"
 $SHARED_SDK_FOLDER="SalesforceMobileSDK-Shared"
-$WINDOWS_SDK_REPO_PATH="https://github.com/forcedotcom/SalesforceMobileSDK-Windows.git"
+$WINDOWS_SDK_REPO_PATH="https://github.com/kchitalia/SalesforceMobileSDK-Windows.git"
```
Comment (bhariharan) [inline SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/plugins/com.salesforce/tools/update.ps1]: This should be ```forcedotcom```, yeah?

File: SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/windows/bld/pri.resfiles.intermediate (hunk 6 above)
Comment (kchitalia) [inline SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/platforms/windows/build/windows/bld/pri.resfiles.intermediate]: Same here

File: SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/plugins/com.salesforce/tools/update.ps1 (hunk 7 above)
Comment (kchitalia) [inline SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/plugins/com.salesforce/tools/update.ps1]: Thats right. Will fix it.

Comment (bhariharan) [inline SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/plugins/com.salesforce/tools/update.ps1]: Is it merged already? You might need a new pull request.

Comment (kchitalia) [inline SalesforceSDK/Salesforce.Sample.Cordova.AccountManager/plugins/com.salesforce/tools/update.ps1]: Is it possible to revert this pull request?

@bhariharan @smcnulty-sfdc

**Pull request merged**